src/atom_finder/classifier.clj

```diff
@@ -49,6 +49,9 @@
    (ValidatedAtom :reversed-subscript        reversed-subscript-atom?    (default-finder reversed-subscript-atom?))
    (ValidatedAtom :literal-encoding          literal-encoding-atom?      (default-finder literal-encoding-atom?))
    (ValidatedAtom :post-increment            post-*crement-atom?         (default-finder post-*crement-atom?))
+   (ValidatedAtom :comma                     comma-atom?                      (default-finder comma-atom?))
+   (ValidatedAtom :curly-braces              curly-braces-atom?               (default-finder curly-braces-atom?))
```

Comment:
can you rename the functions to match the atom names? comma -> comma-operator and curly-braces -> omitted-curly-braces